#This represent a transformation as axis-angle representation

geometry_msgs/Vector3 axis
float32 angle
